### ISSUE TITLE
leafviz error 

### ISSUE THREAD

**Issue by seyoun209** [davidaknowles/leafcutter#232]
Hi, I am trying to follow exactly the steps from the tutorial but I am getting error message>: 

`> shiny::runApp( launch.browser=TRUE ) 

Listening on http://127.0.0.1:3791
Warning: Error in rename: unused argument (ΔPSI = deltapsi)
  107: filter_intron_table [/work/users/s/e/seyoun/07.Doug_thesis/leafcutter_sp/tool/leafcutter/leafviz/server.R#32]
  104: exprFunc [/work/users/s/e/seyoun/07.Doug_thesis/leafcutter_sp/tool/leafcutter/leafviz/server.R#157]
  103: widgetFunc
  102: ::
htmlwidgets
shinyRenderWidget
  101: func
   88: renderFunc
   87: renderFunc
   83: renderFunc
   82: output$cluster_view
    1: shiny::runApp
TableGrob (2 x 1) "arrange": 2 grobs
  z     cells    name           grob
1 1 (1-1,1-1) arrange gtable[layout]
2 2 (2-2,1-1) arrange gtable[layout]`

I am unsure where I have a problem visualizing the delta psi table in leafcutter. I checked the RData, and it contains deltapsi.

**Comment by jackhump** [comment]
Hi, Leafviz is now a standalone app, can you  try installing it and check if you get the same error there? https://github.com/jackhump/leafviz

**Comment by seyoun209** [comment]
Thanks for the response. 

I have tried it, and I still see the same error I couldn't figure out why. I think it is the same problem, but I generated my RData

`Rscript /leafviz/prepare_results.R \
--meta_data_file  sampleInfo.txt \
--code leafcutter ctrvsfnf_perind_numers.counts.gz \
leafcutter_ds_cluster_significance.txt \
leafcutter_ds_effect_sizes.txt \
ref/gecode_hg38_leafcutter \
--output ctrvsfnf.RData`

and I see the deltapsi values in my RData.

`Warning: Error in rename: unused argument (ΔPSI = deltapsi)
  108: filter_intron_table [#6]
  105: exprFunc [/nas/longleaf/home/seyoun/R/oodlibs/leafviz/application/server.R#94]
  104: widgetFunc
  103: ::
htmlwidgets
shinyRenderWidget
  102: func
   89: renderFunc
   88: renderFunc
   84: renderFunc
   83: output$cluster_view
    2: shiny::runApp
    1: leafviz
TableGrob (2 x 1) "arrange": 2 grobs
  z     cells    name           grob
1 1 (1-1,1-1) arrange gtable[layout]
2 2 (2-2,1-1) arrange gtable[layout]`

**Comment by seyoun209** [comment]
I see the deltapsi  value as below table but I am not sure why I cannot visualize through leafviz.

**Comment by jackhump** [comment]
the error is coming from the rename() function from dplyr. Can you check which version of R and dplyr you're using? I haven't seen this bug before.

**Comment by seyoun209** [comment]
Yes. I use `> packageVersion("dplyr")
[1] ‘1.1.1’
`

**Comment by jackhump** [comment]
huh, that's interesting, I wasn't aware there would be a breaking change. Was it the same version of dplyr in both R installs? That would be good to know.

**Comment by jackhump** [comment]
Hi, I just checked the leafviz standalone app code and that line `(ΔPSI = deltapsi)` isn't in there, I changed it to "dPSI" to avoid dealing with greek letter issues. 

Are you sure that you were using the standalone app version of leafviz with R 4.2.2? I'd like to make sure that it still works with the latest version of R. I'm not able to update to 4.2.2 right now as I'm in the middle of the some projects.
